### PR TITLE
Leian/add image

### LINKED ISSUE #96: Add local timesync instance to dev environment [3]
It would be very useful to have a local instance of the timesync server as part of the development environment.  This would allow for the testing of all functionality, including administrative tasks, without impacting other testers.  It would also facilitate replication of database conditions for bug reports.  Finally, it will allow contributors without direct network access to the primary staging server (folks on airplanes, or overseas, for instance) to run tests.

### PR DESCRIPTION
**Pull request by leian7**
Fixes #96 
## Changes in this PR.

- [X] Adds timesync-node image to flask frontend
- [X] Modifies config.py.dist's default TIMESYNC_URL
- [X] Modifies README
## Testing this PR.

1. Run `docker-compose build`
2. Run `docker-compose up`
3. Go to http://localhost:5000/ and login as "admin" with password "admin"
### Expected Output.

@aaroncohen73 @subnomo @Kennric @LyonesGamer @mathuin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ running the app for the first time
 | `SECRET_KEY`   | The secret encryption key used to encrypt sessions. Make sure that it's hard to guess |
 | `TIMESYNC_URL` | The URL of the TimeSync server that this app will use     |
 
-For a full list of configuration options, check out the documentation
+Note: If you are only intending to run the app in a development environment,
```
Comment (aaroncohen73) [inline README.md]: "in a development environment" could be changed to "in a development environment with Docker" for extra clarity.

**Comment by cthalmann** [comment]
Getting a KeyError when trying to submit times, view times, etc. @aaroncohen73 suggested it was due to trying to access an empty array/list.

**Comment by subnomo** [comment]
I get the same error as @cthalmann, and yes it appears to be an issue with there being no existing times.

+1 to this, I'll create a separate issue for the above problem.

**Comment by aaroncohen73** [comment]
@subnomo I believe the above issue is fixed by #110 as it was caused by the app calling `error_message` on an empty list of objects.

**Comment by subnomo** [comment]
@aaroncohen73 Just tested it, viewing times is fixed, but submitting is not.

**Comment by pop** [comment]
@leian7 This is a really cool PR! It's surprisingly simple, way easier than something similar I was going to try and implement.

I want to give this a +1 but when I go to the `/times/create` I get an **Internal Server Error** (_The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application._)

Everything else seems to work well. This doesn't seem to be a problem in the scope of this PR but it is something we should fix before merging. Really good work though, this is super useful.

**Comment by MaraJade** [comment]
+1

**Pull request merged**